routes/user.js

```diff
@@ -89,5 +94,153 @@ module.exports = [
         reply(Boom.badImplementation(err));
       });
     }
+  },
+
+  {
+    method: 'GET',
+    path: '/user/{user}/catalog.json',
+    config: {
+      tags: ['disablePlugins']
+    },
+    handler: (request, reply) => {
+      const { user } = request.params;
+
+      return Promise.all([
+        User.findOne({
+          _id: user
+        }, {
+          name: 1,
+          website: 1
+        }),
+        Meta.find({
+          user
+        }, {
+          bbox: 1, gsd: 1
```

Comment:
I am not sure but I think the idiomatic way to do projections with Mongoose is with string literals.  http://mongoosejs.com/docs/api.html#find_find
I found the 1s confusing since I normally associate this with sort order.

Reply:
I found the space-delimited string literals mildly alarming, so deferred to MongoDB docs. I'll update that.